apps/studio/components/interfaces/App/FeaturePreview/FeaturePreviewContext.tsx

```diff
@@ -1,12 +1,27 @@
-import { noop } from 'lodash'
-import { PropsWithChildren, createContext, useContext, useEffect, useState } from 'react'
-
 import { LOCAL_STORAGE_KEYS } from 'lib/constants'
 import { EMPTY_OBJ } from 'lib/void'
+import { noop } from 'lodash'
+import { PropsWithChildren, createContext, useContext, useEffect, useState } from 'react'
 import { APISidePanelPreview } from './APISidePanelPreview'
 import { CLSPreview } from './CLSPreview'
+import { SqlEditorTabsPreview } from './SqlEditorTabs'
+import { TableEditorTabsPreview } from './TableEditorTabs'
 
 export const FEATURE_PREVIEWS = [
+  {
+    key: LOCAL_STORAGE_KEYS.UI_TABLE_EDITOR_TABS,
+    name: 'Table Editor tabs',
+    content: <TableEditorTabsPreview />,
+    discussionsUrl: 'https://github.com/orgs/supabase/discussions/20295',
```

Comment:
Adding comment to avoid forgetting about the links.

Reply:
removed for now